components/automate-ui/README.md

```diff
@@ -82,40 +82,6 @@ Reason: Later releases are release candidates; should only be using production-r
 
 Per https://github.com/chef/automate/pull/1867, future versions have made a breaking change so it breaks our code base.
 
-### Package tslint: ^5.20.1
-
-Reason: Trying to install the latest version `npm install tslint@6` reports these warnings
-so maintaining at highest version of 5.x.x:
-
-```text
-npm WARN deprecated tslint@6.0.0: TSLint has been deprecated in favor of ESLint. Please see https://github.com/palantir/tslint/issues/4534 for more information.
-npm WARN codelyzer@5.2.1 requires a peer of tslint@^5.0.0 but none is installed. You must install peer dependencies yourself.
-npm WARN rxjs-tslint@0.1.7 requires a peer of tslint@^5.0.0 but none is installed. You must install peer dependencies yourself.
-npm WARN tslint-defocus@2.0.6 requires a peer of tslint@^5.x but none is installed. You must install peer dependencies yourself.
-```
-
-### Package typescript: ~3.5.3
-
-Reason: Trying to install the latest version `npm install typescript@3.7.4` reports these warnings
-so maintaining at highest version of 3.5.x:
-
-```text
-@angular-devkit/build-angular@0.803.21 requires a peer of typescript@>=3.1 < 3.6 but none is installed.
-@angular/compiler-cli@8.2.14 requires a peer of typescript@>=3.4 <3.6 but none is installed.
-@ngtools/webpack@8.3.21 requires a peer of typescript@>=3.4 < 3.6 but none is installed.
-```
-
-### Package @nguniversal/express-engine: 9.0.0-rc.2
-
-Reason: `ng build` reported this error:
-
-```text
-ERROR in The target entry-point "ngx-cookie" has missing dependencies:
- - express-serve-static-core
-```
-
-Reference: https://github.com/salemdar/ngx-cookie/issues/106
```

Comment:
i've since updated this to 10 when dealing with peer dependency warnings from audit fixes, so this warning has been removed again because it is no longer an issue

Reply:
> To address your question on resolvers: resolvers are distinct from Observables. A global search for `resolve:` will reveal that there are only four resolvers in the code base. You simply need to check the `resolve` method in each of those resolvers to satisfy the checklist item to which you referred above.

Thank you for this explanation - very helpful 🙏